contributors/design-proposals/standardization-of-development-process.md

```diff
@@ -0,0 +1,282 @@
+preferably in a mailing list so that the discussion is saved for future Kubernauts.
+
+It may be very helpful to provide links to experience reports of other Kubernetes
+users and developers within the `Motivation` section in order to provide additional
+context for the necessity of a proposed change. The `Examples` section could be
+used to provide motivating examples for documentation writers looking to teach
+others how to consume the proposed change once implemented. As an open source
+community we must constantly be looking for ways to better serve newcomers and
+a `Mentors` section could be used to help connect interested parties with elders
+in the community who are able to mentor people towards an implementation.
+
+It is intended for RFCs to be a conversation, the comment part of the acronym,
+so it is not expected for every section of the template to be completed by the
+time an RFC is filed, however, the `Summary`, `Motivation`, and `Examples` section
+should probably be filled out by the time an RFC is filed. Merging an RFC only
+signals agreement of the sections which have been provided thus far and an RFC
+may be completed through a few rounds of review in order to achieve alignment.
+
+### Git and GitHub Implementation
+
+Practically an RFC would be implemented as a pull request to a central repository
+with the following example structure
+
+```
+├── 0000-rfc-template.md
+├── CODEOWNERS
+├── index.md
+├── sig-architecture
+├── sig-network
+│   └── kube-dns
+├── sig-node
+│   └── kubelet
+├── sig-release
+├── sig-storage
+├── unsorted-to-be-used-by-newcomers-only
+└── wg-resource-management
+```
+
+where each SIG or working group is given a top level directory with components
+maintained by the SIG listed in sub directories. For newcomers to the community
+an `unsorted-to-be-used-by-newcomers-only` directory may be used before an RFC
+can be properly routed to a SIG although hopefully if discussion for a potential
+RFC begins on the mailing lists proper routing information will be provided to
+the RFC author. Additionally a top level index of RFCs may be helpful for people
+looking for a complete list of RFCs. There should be basic CI to ensure that a
+`index.md` remains up to date.
+
+Ideally no work would begin on an RFC before it has been approved by the
+responsible SIG or working group. In order to help combat the explosion of GitHub
+issues against `kubernetes/kubernetes` it is further proposed for each chartered
+SIG to create a top level repository within the `kubernetes` GitHub organization
+where implementation issues would be filed and linked to in the RFC.
+
+### RFC Editor Role
+
+Taking a cue from the [Python PEP process][], I believe that a group of RFC editors
+will be required to make this process successful; the job of an RFC editor is
+likely very similar to the [PEP editor responsibilities][] and will hopefully
+provide another opportunity for people who do not write code daily to contribute
+to Kubernetes.
+
```

Comment:
I can't speak directly to how effective the PEP Editor role has been in their community but perhaps @jbeda could help investigate the effectiveness of the role as he's expressed a good deal of admiration for the PEP process in our conversations.